spec/ship_spec.rb

```diff
@@ -1,19 +1,49 @@
-require 'spec_helper'
+require_relative 'spec_helper'
```

Comment:
I have to look over these helpers again for clarity on usage, but good job getting it to work.